pkgdown/build.R

```diff
@@ -0,0 +1,111 @@
+#' @param repo The path to the git repository to build.
+#' @param versions A list of lists. Each sublist should contain the following keys:
+#'   - `ref`: The ref to build.
+#'   - `href`: The URL path for the version.
+#'   - `label`: The label to display in the navbar. To use the version from DESCRIPTION provide `TRUE`.
+#' Additonally, exactly one version should have `href` set to "/".
+#' @param url The rool URL for all versions of the website.
+#' @param destination The destination directory for the built website.
+build_versioned <- function(repo, versions, url, destination) {
```

Comment:
Would aid readability IMO.
```suggestion
build_versioned <- function(repo, versions, root_url, destination) {
```